macros/PGbasicmacros.pl

```diff
@@ -1410,19 +1410,13 @@ =head2 Display Macros
               Latex2HTML => "output this in Latex2HTML mode",
              )
 
-    TEX     (tex_version, html_version) #obsolete
-
     M3      (tex_version, latex2html_version, html_version) #obsolete
 
 =cut
 
-sub TEX {
```

Comment:
In https://github.com/rederly/pg/pull/1/files you commented out the old line, here you removed them.
Let's keep the change consistent one way or the other.

Reply:
This was also a recommendation, removal > deprecated code clutter.